protocol/message_validator_test.go

```diff
@@ -72,13 +87,15 @@ func (s *MessageValidatorSuite) TestValidateRequestAddressForTransaction() {
 
 func (s *MessageValidatorSuite) TestValidatePlainTextMessage() {
 	testCases := []struct {
-		Name    string
-		Valid   bool
-		Message protobuf.ChatMessage
+		Name             string
+		WhisperTimestamp uint64
+		Valid            bool
```

Comment:
I know this is not directly related to this PR (other than the fact that this PR added more error types), but instead of true/false, should we compare with the actual expected error message?

Reply:
I don't mind either approaches, basically `false` here is to say: `This is not to be validated anymore`, while `Error` means that it can be retried. Probably not in this PR, but you think matching the errors explicitly is a better way? If so I can push the change in a separate PR.
thanks!